test/e2e_node/kubelet_test.go

```diff
@@ -44,12 +44,14 @@ var _ = Describe("Kubelet", func() {
 	})
 
 	Describe("pod scheduling", func() {
+		namespace := "pod-scheduling"
```

Comment:
So the main e2e framework creates new namespaces on the fly. Is that something that will be useful for us? Cleaning up will be as simple as deleting that namespace..